schema.graphql

```diff
@@ -53,7 +53,9 @@ type SendPacket @entity {
     polymerGas: Int
     polymerBlockNumber: BigInt
 
-    feesDeposited: [SendPacketFeeDeposited!] @derivedFrom(field: "sendPacket")
```

Comment:
I disagree with this decision to not track all the fees associated with the packet. Adding lastFeeDeposited as a separate convenience field is fine but it shouldn't prevent us from tracking all the data.